Gulpfile.js

```diff
@@ -265,6 +273,64 @@ gulp.task('test-client-travis', ['build'], function () {
 
 gulp.task('travis', [process.env.GULP_TASK || '']);
 
+//Documentation
+gulp.task('lint-docs', function () {
+    function lintFiles (files) {
+        return new Promise(function (resolve, reject) {
+            var config = {
+                'files':  files,
+                'config': require('./.markdownlint.json')
+            };
+
+            markdownlint(config, function (err, result) {
+                var lintErr = err || result && result.toString();
+
+                if (lintErr)
+                    reject(lintErr);
+                else
+                    resolve();
+            });
+        });
+    }
+
+    return globby('docs/articles/**/*.md').then(lintFiles);
+});
+
+gulp.task('clean-website', function (cb) {
+    del('website', cb);
+});
+
+gulp.task('fetch-assets-repo', ['clean-website'], function (cb) {
+    git.clone('https://github.com/VasilyStrelyaev/testcafe-gh-page-assets.git', { args: 'website' }, cb);
+});
+
+gulp.task('put-in-articles', ['fetch-assets-repo'], function () {
+    return gulp.src('docs/articles/**/*')
+		.pipe(gulp.dest('website/src'));
+});
+
+gulp.task('put-in-navigation', ['fetch-assets-repo'], function () {
+    return gulp.src('docs/_data/**/*')
```

Comment:
Do we really need to follow jekyll's weird naming here. I think it will be better to keep this in `docs/nav` instead of `docs/_data`. Then on build step we can copy it to the appropriate dir.